### PR TITLE
added invertase to the company list

### PR DESCRIPTION
**Pull request by tanmoy27112000**
Thanks for contributing to the [Hacktoberfest Swag List](https://hacktoberfestswaglist.com/) :smiley: :tada:! Before submitting your pull request, please check off as many of the items below as you can:

1. [X] I have read the [Contributing.md](https://github.com/crweiner/hacktoberfest-swag-list/blob/master/CONTRIBUTING.md) file and formatted this PR correctly
2. [X] I'm not adding a company from the blocklist
3. [X] I make sure to fix things promptly if an error or suggestion comes up

Thanks and Happy Hacktoberfest! :tada:
Tagging @crweiner to take a look. :eyes:

### CODE REVIEW

File: index.md [hunk 1]
```diff
@@ -315,6 +315,22 @@ See [**Contributing.md**](./CONTRIBUTING.md) to see how to format your pull requ
 - **How to sign up**: Contribute one PR or participate in the [Hasura Tips & Tricks contest](https://hasura.io/blog/celebrating-open-source-with-hasura/)
 - **Notes**: This year, Hasura is offering two ways to earn their exclusive swags - 1. Fix at least on issue labeled "Hacktoberfest". 2. Participate in their Hasura Tips & Tricks contest on Twitter. [More details here](https://hasura.io/blog/celebrating-open-source-with-hasura/). You only have until November 14th to earn Hasura's unique swag.
 
+### I
+
+#### **INVERTASE**
+
+- **Swag**: Dash, Zappy, Google Home, FlutterVikings rubber duck, Flutter mug, T-shirt, Stickers pack
+- **Requirements**:
+  - Submit 1 or more pull requests and get them approved/merged with tag <strong>hacktoberfest-special</strong> to receive a Dash ,Google Home, Zappy, FlutterVikings rubber duck, Flutter mug, T-shirt, stickers pack.
```
[the remark this answers is not in the document]
Comment (tanmoy27112000) [reply index.md]: resolved by removing inline HTML

File: index.md [hunk 2]
```diff
@@ -315,6 +315,22 @@ See [**Contributing.md**](./CONTRIBUTING.md) to see how to format your pull requ
...
+  - Submit 10 or more pull requests and get them approved/merged with tag <strong>hactoberfest-accepted</strong> to receive a Google Home, Zappy, FlutterVikings rubber duck, Flutter mug, T-shirt, stickers pack.
+  - Submit 5 to 10 pull requests and get them approved/merged with tag <strong>hactoberfest-accepted</strong> to receive a Zappy, FlutterVikings rubber duck, Flutter mug, T-shirt, stickers pack.
```
[the remark this answers is not in the document]
Comment (tanmoy27112000) [reply index.md]: resolved by removing inline HTML

File: index.md [hunk 3]
```diff
@@ -315,6 +315,22 @@ See [**Contributing.md**](./CONTRIBUTING.md) to see how to format your pull requ
...
+  - Submit 4 to 2 pull requests and get them approved/merged with tag <strong>hactoberfest-accepted</strong> to receive a Flutter mug, T-shirt, stickers pack.
```
[the remark this answers is not in the document]
Comment (tanmoy27112000) [reply index.md]: resolved by removing inline HTML

File: index.md [hunk 4]
```diff
@@ -315,6 +315,22 @@ See [**Contributing.md**](./CONTRIBUTING.md) to see how to format your pull requ
...
+  - Submit 1 pull requests and get them approved/merged with tag <strong>hactoberfest-accepted</strong> to receive a T-shirt, stickers pack.
```
[the remark this answers is not in the document]
Comment (tanmoy27112000) [reply index.md]: resolved by removing inline HTML

File: index.md [hunk 5]
```diff
@@ -315,6 +315,22 @@ See [**Contributing.md**](./CONTRIBUTING.md) to see how to format your pull requ
...
 - **Notes**: This year, Hasura is offering two ways to earn their exclusive swags - 1. Fix at least on issue labeled "Hacktoberfest". 2. Participate in their Hasura Tips & Tricks contest on Twitter. [More details here](https://hasura.io/blog/celebrating-open-source-with-hasura/). You only have until November 14th to earn Hasura's unique swag.
 
+### I
+
+#### **INVERTASE**
+
+- **Swag**: Dash, Zappy, Google Home, FlutterVikings rubber duck, Flutter mug, T-shirt, Stickers pack
+- **Requirements**:
+  - Submit 1 or more pull requests and get them approved/merged with tag <strong>hacktoberfest-special</strong> to receive a Dash ,Google Home, Zappy, FlutterVikings rubber duck, Flutter mug, T-shirt, stickers pack.
+  - Submit 10 or more pull requests and get them approved/merged with tag <strong>hactoberfest-accepted</strong> to receive a Google Home, Zappy, FlutterVikings rubber duck, Flutter mug, T-shirt, stickers pack.
```
[the remark this answers is not in the document]
Comment (tanmoy27112000) [reply index.md]: resolved by removing inline HTML

**Comment by crweiner** [comment]
Hello @tanmoy27112000! 👋 Thank you for adding Invertase to the 2022 Hacktoberfest Swag List! 🎉 🎉 🎉 

you rock

I hope we can send lots of meaningful PRs their way. Have a great Hacktoberfest! 🎃

**Comment by tanmoy27112000** [comment]
This was not tagged as Hacktoberfest-accepted. I think this will not count as a valid PR @crweiner

**Comment by crweiner** [comment]
It won't count for Hacktoberfest no matter what I label it because your PR was before Hacktoberfest starts. 😉 PRs start to count as of today because it's October 1st in some parts of the world already. 

Also, as long as the repo is tagged for `Hacktoberfest`, which this one is, merged PRs will count without a `hacktoberfest-accepted` label.

If this is your first Hacktoberfest, I greatly encourage you to [read up on all of the rules and participation guidelines here](https://hacktoberfest.com/participation/#pr-mr-details). Remember, it's not about getting a shirt, it's about learning and making meaningful contributions to open-source projects you care about.

**Comment by tanmoy27112000** [comment]
Got it Thanks @crweiner

**Pull request merged**